client/src/utils/dataFetcher.js

```diff
@@ -101,14 +102,15 @@ const savePatch = async (documentId, delta, userId, quillRef) => {
       body: JSON.stringify(body),
     });
     if (!response.ok) throw new Error("Failed to save patch");
+
+    const result = await response.json();
+    return result;
   } catch (error) {
     console.error("Cannot save patch", error.message);
```

Comment:
Tip for future Dijon: Errors happen, they suck, we should alert the user rather that fail silently

<img width="433" height="159" alt="image" src="https://github.com/user-attachments/assets/ee61e04f-d758-4455-9310-d15435fa2cb3" />